code/modules/paperwork/fax.dm

```diff
@@ -0,0 +1,133 @@
+/obj/machinery/fax
+	name = "Fax Machine"
+	desc = "Bluespace technologies on the application of bureaucracy."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "fax"
+	density = TRUE
+	power_channel = AREA_USAGE_EQUIP
+	max_integrity = 300
+	pass_flags = PASSTABLE
+	/// The unique ID by which the fax will build a list of existing faxes.
+	var/fax_id
+	/// The name of the fax displayed in the list. Not necessarily unique to some EMAG jokes.
+	var/fax_name
+	/// A reference to an `/obj/item/paper` inside the copier, if one is inserted. Otherwise null.
+	var/obj/item/paper/paper_contain
+	/// Necessary to hide syndicate faxes from the general list
+	var/syndicate_network = FALSE
+
+/obj/machinery/fax/Initialize(mapload)
+	. = ..()
+	if (!fax_id)
+		fax_id = SSnetworks.assign_random_name()
+	if (!fax_name)
+		fax_name = "Unregistered fax " + fax_id
+
+/obj/machinery/fax/Destroy()
+	QDEL_NULL(paper_contain)
+	return ..()
+
+/obj/machinery/fax/update_icon_state()
+	if(paper_contain)
+		icon_state = "fax_contain"
+	else
+		icon_state = "fax"
+	return ..()
+
+/obj/machinery/fax/emag_act(mob/user)
+	if(!(obj_flags & EMAGGED))
+		obj_flags |= EMAGGED
+		playsound(src, 'sound/creatures/dog/growl2.ogg', 50, FALSE)
+		to_chat(user, span_warning("An image appears on [src] screen for a moment with Jan in the cap of a Syndicate officer."))
+
+/obj/machinery/fax/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	default_unfasten_wrench(user, tool)
+	return TOOL_ACT_TOOLTYPE_SUCCESS
+
+/obj/machinery/fax/attackby(obj/item/I, mob/user, params)
```

Comment:
please use a better var name than ``I``.

Reply:
> please use a better var name than `I`.

Most of the code uses **/obj/item/I** and I take that as standard.

Reply:
It's not. https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#dont-use-abbreviations